lib/mbta_v3_api/stops/api.ex

```diff
@@ -152,8 +152,7 @@ defmodule MBTAV3API.Stops.Api do
   end
 
   @spec parse_v3_response(Item.t() | {:ok, Item.t()} | {:error, any}) ::
-          {:ok, Stop.t() | nil}
-          | {:error, any}
+          {:ok, any} | {:error, any}
```

Comment:
Would you mind also explaining this?

Reply:
I'm trying to fix all the dialyzer warnings in this repo, in preparation to getting CI running on it, so I added a few to this PR. Honestly I think the return type for this one OUGHT to be `{:ok, Stop.t() | nil} | {:error, any}` but I can't get it to accept that, so I've left it as you see here.